automation/android/espresso/build.gradle

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (C) 2017 - present Instructure, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+buildscript {
+    // Define this once here because it will need to be reused later
+    ext.kotlinVersion = "1.2.70"
```

Comment:
this version of kotlin is now quite old, we'll want to update to the latest stable

Reply:
Does that mean that we would want to up-version Kotlin in android-uno as well, to keep the versions in sync?

Reply:
Ideally yes, we'd want android-uno on the latest Kotlin version. Upgrading android-uno is typically more challenging because there's a lot of code.

We should be able to use the newer version on this lib and still use it from an older Kotlin in android-uno.